pkg/operator/ceph/cluster/osd/osd.go

```diff
@@ -740,3 +761,68 @@ func getOSDInfo(d *apps.Deployment) ([]OSDInfo, error) {
 
 	return []OSDInfo{osd}, nil
 }
+
+func GetLocationWithNode(clientset kubernetes.Interface, nodeName string, crushHostname string) (string, error) {
+
+	node, err := getNode(clientset, nodeName)
+	if err != nil {
+		return "", errors.Wrapf(err, "could not get the node for topology labels")
+	}
+
+	hostNameLabel := crushHostname
```

Comment:
You could remove this line and just rename the var `hostNameLabel` to `crushHostname`. No need to define a new var.